src/amo/components/AddonDetail.js

```diff
@@ -114,9 +114,10 @@ export class AddonDetailBase extends React.Component {
     );
   }
 
-  readReviewsFooter() {
-    const { addon, i18n } = this.props;
+  renderRatingsCard() {
+    const { addon, i18n, RatingManager } = this.props;
```

Comment:
Properties `ThatStartWithCaps` should go first if we're alphabetising props (as we tend to).